_studio/mfx_lib/shared/include/mfx_h264_encode_struct_vaapi.h

```diff
@@ -314,6 +314,20 @@ typedef struct tagENCODE_CAPS
     UCHAR    MaxNum_WeightedPredL1;
 } ENCODE_CAPS;
 
+// We cannot modify ENCODE_CAPS structure, but need to add new features
+struct MFX_ENCODE_CAPS
+{
+    ENCODE_CAPS ddi_caps;
+    union {
+        struct {
+            UINT CQPBRCSupport: 1;
```

Comment:
I think you don't need to be economical with bits here. I suggest to use bool's without union.
The rest is fine.

Reply:
Done.